packages/shorebird_cli/lib/src/commands/release/aar_releaser.dart

```diff
@@ -76,7 +76,8 @@ class AarReleaser extends Releaser {
 
   @override
   Future<FileSystemEntity> buildReleaseArtifacts() async {
-    final flutterVersionString = await shorebirdFlutter.getVersionAndRevision();
+    final flutterVersionString =
+        await shorebirdFlutter.humanReadableVersionFromEnv();
```

Comment:
I'd advocate for reverting the naming change since the previous name was more descriptive and it would dramatically reduce the diff 🤷